### ISSUE TITLE
Jikkyou Oshaberi Parodius having trouble saving and loading the save.

### ISSUE THREAD

**Issue by atds55** [MrPalloncini/PicoMemcard#5]
The problem is similar to what occurs with sd2psx, but in the case of the picomemcard with the new fw, the game does not save the first time, and when I copy a save from another memory card, it takes a while to load or save the game.
It behaves similarly to the videos I posted here: https://github.com/sd2psx/firmware/issues/29

https://github.com/MrPalloncini/PicoMemcard/assets/72902338/46539475-4a15-4042-a0ed-27fcd22ca96a

**Comment by MrPalloncini** [comment]
man, i'm sorry but i don't think i can fix this in the next release, i'm still fixing important issues right now and i need to know how this game write blocks. Are there any other games that gave you issues with saving?

**Comment by atds55** [comment]
> cara, me desculpe, mas acho que não posso consertar isso na próxima versão, ainda estou corrigindo problemas importantes agora e preciso saber como esse jogo bloqueia a gravação. Há algum outro jogo que lhe causou problemas para salvar?

I only found this game with this problem.
There must be something different in the way this game saves, because sd2psx also has a problem with this game. People who have the 8Bitmods memcard pro say that this problem does not occur.
No problem, priority for other more important problems, I created this topic just to report the problem.
Thanks.

**Comment by MrPalloncini** [comment]
Did you tryied the latest sd2psx firmware with that game?

**Comment by atds55** [comment]
Yes, and it still has the same problem.

**Comment by nullpointer-br** [comment]
Using PSIO game was freeze at memory card check/save creation... LE_K1V2 mode solved the problem.

**Comment by atds55** [comment]
> Using PSIO game was freeze at memory card check/save creation... LE_K1V2 mode solved the problem.

Yes, I had mentioned this on the PSIO discord and the PS2 Scene discord, but I don't think you're there. But on XStation there is no k1v2 loadexec mode, so the problem persists.
